jwst/assign_wcs/util.py

```diff
@@ -227,7 +227,7 @@ def is_fits(input):
         isfits = True in [input.endswith(l) for l in names]
 
     # if input is a fits file determine what kind of fits it is
-    #waiver fits len(shape) == 3
+    # waiver fits len(shape) == 3
```

Comment:
Aaack! I don't think we need to worry about wavered FITS format anymore ...

Reply:
:rofl:

Reply:
Just FYI, I'm not changing this part of the code right now, it was copied in for use in some other part of the pipeline (though I don't think the waivered fits logic is being used)